composer.json

```diff
@@ -20,8 +20,7 @@
     "require": {
         "php": ">=8.0",
         "ext-redis": "*",
-        "symfony/framework-bundle": "^5.0|^6.0",
-        "sensio/framework-extra-bundle": "^5.0|^6.0"
+        "symfony/framework-bundle": "^5.0|^6.0"
```

Comment:
It seems that php-cs-fixer v2 can be removed since we have dropped support for Symfony 5.